### PR TITLE
Create displayWikiInfo.php

### PR DESCRIPTION
**Pull request by yaronkoren**
As recommended in issue #114.

### CODE REVIEW

**Comment by yaronkoren** [comment]
I've tested this in a regular wiki instance but not within Canasta; so I'm guessing that at least the path on line 32 is incorrect.

File: Dockerfile [hunk 1]
```diff
@@ -677,6 +677,7 @@ COPY _sources/configs/php_error_reporting.ini _sources/configs/php_upload_max_fi
 COPY _sources/configs/php_max_input_vars.ini _sources/configs/php_max_input_vars.ini /etc/php/7.4/apache2/conf.d/
 COPY _sources/configs/php_timeouts.ini /etc/php/7.4/apache2/conf.d/
 COPY _sources/scripts/*.sh /
+COPY _sources/scripts/*.php $MW_HOME/maintenance/
```
Comment (jeffw16) [inline Dockerfile:680]: Should we place this in an extension in something like `$MW_HOME/extensions/Canasta/maintenance/`?

like this https://github.com/samwilson/RenderThumbnails/

**Review by jeffw16 — APPROVED** [review]
Approving but see comments

File: Dockerfile (hunk 1 above)
Comment (jeffw16) [reply Dockerfile:680]: Let's worry about this later

**Pull request merged**